### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by kingofguan** [aaparashar/racial-bias-detection#14]
This project studied textual data by sports commentators that detect potential racial biases. The data set include commentators’ transcripts from YouTube with the player’s racial identity as a label. The team vectorize the textual data, see if the text differs for a player of different races, and implemented different models to choose the optimal predictor.
 
**Areas of Strengths:**
 
- I am impressed by how the team specified the problem by turning it into a classification task. When I first thought of detecting racial bias, my immediate question was how to label tweets with a racial bias to produce a massive, meaningful dataset. Therefore, the team’s design of detecting certain language implying certain racial identities were very well designed.
 
- The team’s effort of filtering out the effect of positions proved reflected excellent knowledge about the problem and data structures. The filtering could also make the results more interpretable because the position feature provides information about the player’s style, requirements, and contribution. 
 
- The team’s word embedding demonstrated great methods for complex data processing. Textual data are very complicated because of differences in length, format, etc. Vectorizing the data is necessary to fit the model. Also, the team used pre-trained models. That is a good example of transfer learning that we can use to solve relevant problems without starting from scratch every time.
 
**Areas for improvements/Advice:**
 
- The report does not analyze what was special about textual data beyond vocabulary, such as the structure of a sentence, grammar, tokens, etc. I believe additional coverage about textual data could help extract more structural information about the dataset and make the result more interpretable.
 
- I think the report could get even better if it includes introductions about how embedding methods such as TF IDF, FastText, and GloVe differs. Specifically, I think it is important to state the assumptions these embedding methods made and how their performances would differ in different problems. 
 
- I would like to see more explanation about why random forests would overfit. Base on my understanding of Natural Language Processing, a complex model such as a random forest or neural net, is often required to capture complex or long-distance relationships in the data. I believe some justification about the overfitting issue could help.
 
Still, given the projects' page limits, I believe the report had covered critical points about the problem. I took away a lot from reading it. Have a great holiday!
